theme/js/main.js

```diff
@@ -1,3 +1,11 @@
 const {message} = require("./_module");
 
 console.log(message);
+
+// Mobile Menu
+const mobMenu = document.querySelector("#mobile-nav");
+mobMenu.addEventListener("change", e =>
+{
+  const data = document.querySelector(".data");
+  data.textContent = `Data to be added here.`;
```

Comment:
I think when we start learning JavaScript, things like this will become more intuitive for you. Currently this code doesn't make a redirection. This code says "Select element with class `.data` and add `Data to be added here.` string as it's text content". I think here you want to find out which item from the `<select>` element was selected by user and navigate to that specific page. As [was mentioned here](https://github.com/browser-automation/cba-website/pull/7/files#r409189013) you could check a similar example.

You might want here to do the following after the [change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) has been fired:
- Get the selectbox element
- Find out the changed value of the selectbox.
- Navigate to the selected page according to the changed value.

First you need to get the selectbox element, in fact you have it defined above in the line: `const mobMenu = document.querySelector("#mobile-nav"); `, where you have used [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) method which can help you find an element using CSS selector.

You can then use [select element's value attribute](https://html.spec.whatwg.org/multipage/form-elements.html#dom-select-value) which returns value of the first selected options element, as we don't have multiselect selectbox here, we can rely on that result. After you have the value, the only thing is to navigate to the page specified in the value.

In order to navigate the page according to the user selection in the selectbox we can use [window](https://developer.mozilla.org/en-US/docs/Web/API/Window) object's [location](https://developer.mozilla.org/en-US/docs/Web/API/Window/location) property. In the example I have posted above, we have used that property to create an absolute URL, from protocol, domain and pathname, but the thing is that we can do it even more simpler by modifying [window.location.pahtname](https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname), which actually contain the pathname of the page you are located in, by changing the pathname you will also make a redirection.